buddyboss/buddyboss-platform
# PROD-684 fixed #3177 autocomplete suggestions position changed to make it near the searchbox

PROD-684 #comment autocomplete position changed to fix the suggestion dropdown to make it near the search box

_The min width is set to make the design clear and suggestions to be clearly visible so the point no 1 in the issue #3177 is intentionally set with min width and doesn't require any change_

**Fixes** suggestions list  loads far way from the actual search box in two cases-
1. When the search box is at the bottom right side of the screen and no space at bottom of the screen, the suggestion list loads far to the left of the search box and half out of viewport
2. In the RTL mode the suggestion list fully invisible as its completely out of the viewport

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3177 .

### How to test the changes in this Pull Request:

- Before checking PR, create a page using elementor with the sample content
- Add another 3 column row with the  bb forum search widgets ( in first and third column ) such that there is not space at bottom to load the suggestion list
- View page, type any character on the last search box, see that the suggestion result is loaded far away from the search box
- Go to settings > general, change to any RTL language
- View page again, type any character on the search box, see that the suggestion result box is placed most right. Please note that you may also not see the suggestion box as it is out of the screen due to the current styling
- Now check the same with the PR it fixes both the issue

### Proof Screenshots or Video


https://screencast-o-matic.com/watch/c3V16kVoFfM

### Other information:




### Changelog entry

> Autocomplete suggestions position changed to make it near the searchbox
